WORKSPACE

```diff
@@ -594,3 +598,20 @@ debian_deps()
 load("@bazel_skylib//:workspace.bzl", "bazel_skylib_workspace")
 
 bazel_skylib_workspace()
+
+load("@rules_jvm_external//:repositories.bzl", "rules_jvm_external_deps")
```

Comment:
You may asa well get ahead of buildifier lint and put in the blank lines between each.